packages/gatsby/cache-dir/static-entry.js

```diff
@@ -260,7 +264,27 @@ export default async function staticPage({
     // If no one stepped up, we'll handle it.
     if (!bodyHtml) {
       try {
-        bodyHtml = renderToString(bodyComponent)
+        // react 18 enabled
+        if (pipeToNodeWritable) {
+          const {
+            WritableAsPromise,
+          } = require(`./server-utils/writable-as-promise`)
+          const writableStream = new WritableAsPromise()
+          const { startWriting } = pipeToNodeWritable(
+            bodyComponent,
+            writableStream,
+            {
+              onCompleteAll() {
+                startWriting()
+              },
+              onError() {},
```

Comment:
Do we want to ignore errors here?

Reply:
we won't do streaming yet until libs have adopted and maybe we never do.
React still throws the error so this is just to don't show the warning.